webrender/src/platform/unix/font.rs

```diff
@@ -613,10 +663,17 @@ impl FontContext {
         }
 
         match format {
+            FT_Glyph_Format::FT_GLYPH_FORMAT_BITMAP => {
+                if font.flags.contains(FontInstanceFlags::SYNTHETIC_ITALICS) {
+                    let (skew_width, skew_left) = skew_bitmap(&mut final_buffer, actual_width, actual_height, left, top);
```

Comment:
the function returns some stuff and mutates the other. It would be cleaner to just return all the results, as in:
```rust
let (new_buffer, skew_width, skew_left) = skew_bitmap(final_buffer, ...);
```